### ISSUE TITLE
unable to compile

### ISSUE THREAD

**Issue by lyoni** [rauenzi/BDPluginLibrary#145]
unable to compile

`TypeError: Cannot read property 'AccessibilityPreferencesContext' of undefined
    at Object../src/ui/settings/settingfield.js (betterdiscord://plugins/0PluginLibrary.plugin.js:5157:128)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at Object../src/ui/settings/index.js (betterdiscord://plugins/0PluginLibrary.plugin.js:5107:71)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at Object../src/ui/ui.js (betterdiscord://plugins/0PluginLibrary.plugin.js:6441:67)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at Object../src/modules/pluginupdater.js (betterdiscord://plugins/0PluginLibrary.plugin.js:1832:60)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at Object../src/structs/plugin.js (betterdiscord://plugins/0PluginLibrary.plugin.js:3866:80)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at Object../src/structs/structs.js (betterdiscord://plugins/0PluginLibrary.plugin.js:4077:65)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at Object../src/modules/domtools.js (betterdiscord://plugins/0PluginLibrary.plugin.js:627:65)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at Object../src/modules/modules.js (betterdiscord://plugins/0PluginLibrary.plugin.js:1538:67)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6484:41)
    at eval (betterdiscord://plugins/0PluginLibrary.plugin.js:6531:65)
    at eval (betterdiscord://plugins/0PluginLibrary.plugin.js:6580:3)
    at eval (betterdiscord://plugins/0PluginLibrary.plugin.js:6583:12)
    at Object.requireAddon (<anonymous>:4:43438)
    at Object.loadAddon (<anonymous>:4:9197)
    at Object.loadAddon (<anonymous>:4:41165)
    at AsyncFunction.<anonymous> (<anonymous>:4:7353)`

**Comment by thetablespoon** [comment]
the same thing is happening to me

**Comment by kubsakkk** [comment]
me too
